common/util/FontUtils.cpp

```diff
@@ -320,6 +345,47 @@ std::string GameTextFontBank::convert_game_to_utf8(const char* in) const {
   return replace_to_utf8(result);
 }
 
+/*!
+ * Same as convert_game_to_utf8, but gets rid of all special encodings like <COLOR_WHITE>.
+ */
+std::string GameTextFontBank::convert_game_to_utf8_special(const char* in) const {
+  std::string temp;
+  std::string result;
+  std::string test = in;
+  while (*in) {
+    auto remap = find_encode_to_utf8(in);
+    if (remap != nullptr) {
+      temp.append(remap->chars);
+      in += remap->bytes.size() - 1;
+    } else if (valid_char_range(*in) || *in == '\n' || *in == '\t' || *in == '\\' || *in == '\"') {
+      temp.push_back(*in);
+    } else {
+      temp += fmt::format("\\c{:02x}", uint8_t(*in));
+    }
+    in++;
+  }
+  replace_to_utf8_special(temp);
+  for (size_t i = 0; i < temp.length(); ++i) {
+    auto c = temp.at(i);
+    if (c == '\n') {
+      result += "\\n";
+    } else if (c == '\t') {
+      result += "\\t";
+    } else if (c == '\\') {
+      if (i < temp.length() - 1 && temp.at(i + 1) == 'c') {
+        result.push_back(c);
+      } else {
+        result += "\\\\";
+      }
+    } else if (c == '"') {
+      result += "\\\"";
+    } else {
+      result.push_back(c);
+    }
+  }
+  return replace_to_utf8_special(result);
+}
+
```

Comment:
Do you really need all this? Surely you can just `convert_game_to_utf8` and then erase everything inbetween < and >? Would avoid having even more duplicate code here.